packages/common/src/index.ts

```diff
@@ -442,9 +448,8 @@ export default class Common extends EventEmitter {
   hardforkGteHardfork(
     hardfork1: string | null,
     hardfork2: string,
-    opts?: hardforkOptions
+    opts: hardforkOptions = {}
   ): boolean {
-    opts = opts !== undefined ? opts : {}
     const onlyActive = opts.onlyActive === undefined ? false : opts.onlyActive
     hardfork1 = this._chooseHardfork(hardfork1, opts.onlySupported)
```

Comment:
Side note, just because I am seeing it here: we might want to consider to remove onlySupported and/or onlyActive HF options on the next breaking release - this is not really used (maybe in the VM, but I would think this is rather not necessary to be managed in Common) - have added a reminder note to v6 Release planning.